mrrc/storage/s3client.py

```diff
@@ -193,6 +193,19 @@ def get_files(self, bucket_name=None, prefix=None, suffix=None)-> List[str]:
             files = [i.key for i in objs]    
         return files
 
+    def download_file(self, bucket_name=None, key=None, download_path=None):
```

Comment:
Maybe something you need is just read the file content but not download it. You can use these code to do it:
fileObject = bucket.Object(path)
return str(fileObject.get()['Body'].read(), 'utf-8')